pandas/_libs/groupby.pyx

```diff
@@ -1,16 +1,17 @@
 # cython: profile=False
 
-from numpy cimport *
 cimport numpy as np
```

Comment:
we're inconsistent about this, but at least I prefer the `cimport numpy as np`, so it is clearer which parts need c-level access

Reply:
agree here

Reply:
Coming right up.